components/productCard/ProductImage.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="product-image">
+    <a :href="url" :title="title">
+      <img width="200" height="200" :src="cloudinaryImg" :alt="title" />
+    </a>
+  </div>
+</template>
+
+<script lang="ts">
+import Component from 'vue-class-component'
+import { Prop, Vue } from 'vue-property-decorator'
+
+@Component({})
+class ProductImage extends Vue {
+  @Prop(String) imageUrl: string
+  @Prop(String) title: string
+  @Prop(String) url: string
```

Comment:
What is this? It's not like

```
props: {
  imageUrl: String,
  title: String,
  url: String,
```

Reply:
I had to do that to get along with `@Component`

Reply:
😎